hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -166,10 +166,10 @@
                cov: true
              }
              {
-               name: rv_plic_fpv
-               fusesoc_core: lowrisc:fpv:rv_plic_fpv
+               name: top_earlgrey_rv_plic_fpv
+               fusesoc_core: lowrisc:opentitan:top_earlgrey_rv_plic_fpv
```

Comment:
the middle portion here "opentitan" seems a bit different from before.

Reply:
Yes, it's different, but intentional: these IP blocks are specific to OpenTitan, they are not generic lowRISC IP. I therefore put them into the "opentitan" library (that's what the second part of the Vendor-Library-Name-Version (VLNV) string stands for).

Reply:
i guess the part where this is confusing to me is because all the stuff under `hw/ip` and `hw/ip_autogen` are all under the `opentitan` repo. So in a way it feels like they are all specific to opentitan. 

I actually thought you would want something like `lowrisc:top_earlgrey:rv_plic_fpv` to show their specificity to the top level while maintaining a more generic function.  What do you think?

Reply:
The "lowrisc" namespace is actually for all things lowRISC produces, which is more than OpenTitan. We for example share IP blocks with Ibex and other things we're (now or in the future are/might be) working on.

We had for a long time plans to split out the "common" lowRISC IP into a separate repository (e.g. in discussions with Scott and Mark ages ago). We never followed through with that due to the churn it would create for the OpenTitan project. Instead, we went for using the opentitan repository as "source", and vendor the IP blocks into other repositories, e.g. Ibex ([here](https://github.com/lowRISC/ibex/tree/master/vendor)).

So while it is the most ergonomic choice right now to keep all IP in the opentitan repository, I'd like to use the opportunity to get the naming and namespacing more aligned with this vision and avoid a potential massive renaming down the line.

Reply:
ah okay that's a fair point. The common modules being inside the `opentitan` repo is a bit odd and i imagine will cause issues elsewhere as well.

Reply:
btw @imphil , let's say in the long term world we split `hw/ip` into its own common repository, would the `hw/ip_template` stuff also go there? Technically it feels like it should right? Those are not necessarily specific to `opentitan`.

Reply:
Yes, that would be my current take on it.

Reply:
yeah that sounds good and very consistent. 
thanks!